### ISSUE TITLE
[Question] How to change admin password in Gravitee AM?

### ISSUE THREAD

**Issue by adubreuiltk** [gravitee-io/issues#2694]
Gravitee AM works fine with the default admin password (`adminadmin`).

However, I am unable to change it:
- I do not see any option in Global Settings to change the admin password (I can see the the user `admin`, but not option to change/reset the password)
- In the gravitee.yml config file, I tried to change the config, but it did not work :

```yaml
services:
  core:
    http:
      enabled: true
      port: 18092
      host: localhost
      authentication:
        type: basic
        users:
          admin: my-new-password # instead of 'adminadmin'

```
- I even stopped Gravitee AM, dropped the MongoDB database `gravitee-am` and restarted Gravitee AM &rarr; Gravitee successfully started, but the default `adminadmin` password remains.

### Expected Behavior
An option to change the default admin password is welcome.

### Possible Solution
Option to change the admin password in UI

### Steps to Reproduce (for bugs)
See above

### Your Environment
* Version used: Gravitee AM 2.9.0
* Browser Name and version: Chrome 77
* Operating System and version: Windows 10

**Comment by tcompiegne** [comment]
Hi @adubreuiltk ,

The default admin account is stored in the `Inline users` identity provider : 

`https://AM_UI/settings/management/providers`

**Comment by adubreuiltk** [comment]
Thanks @tcompiegne 

When I try to edit any identify provider, I got the error "**Domain [null] can not be found.**":
![image](https://user-images.githubusercontent.com/5410883/66047847-c9497d80-e528-11e9-8f83-bcc0fbe284f8.png)
![image](https://user-images.githubusercontent.com/5410883/66047855-ce0e3180-e528-11e9-85f4-c758c86a28d6.png)

Note: I created a domain.

Any idea?

**Comment by brasseld** [comment]
It has been fixed in 2.8.1 but not yet release

See https://github.com/gravitee-io/issues/issues/2677

**Comment by adubreuiltk** [comment]
Thanks. I'll wait for the release.
